Explorer/Assets/Scripts/Ipfs/Ipfs.cs

```diff
@@ -0,0 +1,66 @@
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+using UnityEngine.Networking;
+public class Ipfs
+{
+    public class ContentDefinition
+    {
+        public string file;
+        public string hash;
+    }
+
+    public class EntityDefinition : EntityDefinitionGeneric<object> {}
+
+    public class SceneEntityDefinition : EntityDefinitionGeneric<SceneMetadata> {}
+
+    public class EntityDefinitionGeneric<T>
+    {
+        public string id;
+        public string[] pointers;
```

Comment:
Most probably fields should be changed to `List` and instantiated to be able to parse to the same instances, but it should be validated with the profiler